pkg/model/resource/resource_test.go

```diff
@@ -111,33 +89,6 @@ var _ = Describe("Resource", func() {
 
 			resource = options.NewResource(multiGroupConfig, true)
 			Expect(resource.Plural).To(Equal("firstmates"))
-
-			resource = options.NewV1Resource(v1Config, true)
-			Expect(resource.Plural).To(Equal("firstmates"))
-
-			options = &Options{Group: "crew", Version: "v1", Kind: "Fish"}
-			Expect(options.Validate()).To(Succeed())
-
-			resource = options.NewResource(singleGroupConfig, true)
-			Expect(resource.Plural).To(Equal("fish"))
-
-			resource = options.NewResource(multiGroupConfig, true)
-			Expect(resource.Plural).To(Equal("fish"))
-
-			resource = options.NewV1Resource(v1Config, true)
-			Expect(resource.Plural).To(Equal("fish"))
-
-			options = &Options{Group: "crew", Version: "v1", Kind: "Helmswoman"}
-			Expect(options.Validate()).To(Succeed())
-
-			resource = options.NewResource(singleGroupConfig, true)
-			Expect(resource.Plural).To(Equal("helmswomen"))
-
-			resource = options.NewResource(multiGroupConfig, true)
-			Expect(resource.Plural).To(Equal("helmswomen"))
-
-			resource = options.NewV1Resource(v1Config, true)
-			Expect(resource.Plural).To(Equal("helmswomen"))
```

Comment:
Some of these tests are not v1. If you watch carefully, all the tests have 4 pair of lines, the first pair creates the options and validates it, the second creates a resource for a single-group v2 project, the third creates it for a multi-group v2 project and the forth creates it for a v1 project. So basically only the pairs where `NewV1Resource(v1Config, ...)` is being used should be removed, the rest need to stay.

Reply:
It is true good catch.

Reply:
You can mark this as solved.